utils/hct/hctdb_instrhelp.py

```diff
@@ -1535,11 +1537,22 @@ def get_interpretation_table():
     gen = db_sigpoint_gen(db)
     return run_with_stdout(lambda: gen.print_interpretation_table())
 
+# highest recognized is different than highest released,
+# since there can be prerelease versions that are higher
+# than the last released version
+highest_recognized_major = 6
+highest_recognized_minor = 9
```

Comment:
I don't think we needed to rename these to add "recognized".  When I was talking to you about "highest recognized shader model", that's what these variables already defined.  By renaming this variable at the same time as introducing the new one, it makes it harder to tell exactly what's different.